go.mod

```diff
@@ -4,39 +4,38 @@ go 1.20
 
 replace (
 	github.com/knative/pkg => github.com/rancher/pkg v0.0.0-20190514055449-b30ab9de040e
-	k8s.io/client-go => k8s.io/client-go v0.27.4
 	sigs.k8s.io/json => sigs.k8s.io/json v0.0.0-20211208200746-9f7c6b3444d2
 )
 
 require (
-	github.com/Masterminds/sprig/v3 v3.2.3
-	github.com/apparentlymart/go-cidr v1.1.0
-	github.com/aws/aws-sdk-go v1.44.272
+	github.com/Masterminds/sprig/v3 v3.2.2
+	github.com/apparentlymart/go-cidr v1.0.1
+	github.com/aws/aws-sdk-go v1.38.65
 	github.com/blang/semver v3.5.1+incompatible
-	github.com/coreos/go-semver v0.3.1
+	github.com/coreos/go-semver v0.3.0
 	github.com/docker/distribution v2.8.2+incompatible
-	github.com/docker/docker v24.0.2+incompatible
+	github.com/docker/docker v20.10.25+incompatible
```

Comment:
Discussed with @vardhaman22 offline, this is okay as k8s is also on v20.10.x for now - [link](https://github.com/kubernetes/kubernetes/blob/a42f4f61c2c46553bfe338eefe9e81818c7360b4/CHANGELOG/CHANGELOG-1.27.md?plain=1#L1377)